idkit/src/store/idkit.ts

```diff
@@ -109,11 +110,20 @@ const useIDKitStore = create<IDKitStore>()((set, get) => ({
 		}: Config,
 		source: ConfigSource
 	) => {
+		const sanitized_credential_types = credential_types?.filter(type => {
+			const isValid = Object.values(CredentialType).includes(type)
+			if (!isValid) {
+				console.warn(`Ignoring invalid credential type received: \`${type}\`.`)
+			}
+			return isValid
+		})
+
```

Comment:
Typescript should take care of this? Unless the app segfaults with invalid credentials we should just let the user "suffer the consequences of their actions" imo